--- ./Makefile.org	Sat Jun 16 16:29:32 2001
+++ ./Makefile	Sat Jul 13 17:52:51 2002
@@ -40,9 +40,9 @@
 	rm osd_mail-$(VERSION)
 
 install: all
-	mkdir -p ${BINDIR}
-	$(INSTALL) osd_mail $(BINDIR)
-	$(INSTALL) osd_mailplus $(BINDIR)
+	$(INSTALL) -d $(DESTDIR)${BINDIR}
+	$(INSTALL) osd_mail $(DESTDIR)$(BINDIR)
+	$(INSTALL) osd_mailplus $(DESTDIR)$(BINDIR)
 
 clean:
 	rm -f *~ *.o *.o.pic osd_mail osd_mailplus
